docs/modules/ROOT/pages/discovery.adoc

```diff
@@ -0,0 +1,57 @@
+:clusterName: simple-derby-druid
+:namespace: stackable
+:routerPort: 8888
+
+= Discovery Profiles
+
+The Stackable Operator for Apache Druid creates a single discovery profile that allows access to multiple Druid services which are published into the Kubernetes cluster as a
```

Comment:
> single discovery profile

What is a discovery profile? I'm confused (as a user)

> multiple Druid services

such as? I'd give some exmaples here (the router console, the SQL endpoint, ...)

Reply:
In the ZK docs there's this part of the sentence "which are client configuration bundles" which I would include here too, to explain what a Discovery Profile is. Maybe - since we only have a single one here, we could instead have the title of the document as "Discovery Configuration Bundle"